draft-ietf-quic-multipath.md

```diff
@@ -1120,33 +1118,69 @@ Destination Connection ID Sequence Number:
 
 Path Status sequence number:
 : A variable-length integer specifying
-  the sequence number assigned for this PATH_STATUS frame. The sequence
+  the sequence number assigned for this PATH_STANDBY frame. The sequence number space is shared with the PATH_AVAILABLE frame and
+the sequence
   number MUST be monotonically increasing generated by the sender of
-  the PATH_STATUS frame in the same connection. The receiver of
-  the PATH_STATUS frame needs to use and compare the sequence numbers
+  the PATH_STANDBY frame in the same connection. The receiver of
+  the PATH_STANDBY frame needs to use and compare the sequence numbers
   separately for each Destination Connection ID Sequence
   Number.
 
-Available values of Path Status field are:
+Frames may be received out of order. A peer MUST ignore an incoming
+PATH_STANDBY frame if it previously received another PATH_STANDBY frame
+or PATH_AVAILABLE 
+for the same Destination Connection ID Sequence Number with a
+Path Status sequence number equal to or higher than the Path Status
+sequence number of the incoming frame.
+
+PATH_STANDBY frames are ack-eliciting. If a packet containing a
+PATH_STANDBY frame is considered lost, the peer SHOULD resend the frame
+only if it contains the last status sent for that path -- as indicated
+by the sequence number.
+
 
-- 1: Standby
-- 2: Available
+## PATH_AVAILABLE frame {#path-available-frame}
 
-Endpoints use PATH_STATUS frame to inform the peer whether it prefer to
-use this path or not. If an endpoint receives a PATH_STATUS frame
-containing 1-Standby status, it SHOULD stop sending non-probing packets
-on the corresponding path, until it receives a new PATH_STATUS frame
-containing 2-Available status with a higher sequence number referring to
-the same path.
+PATH_AVAILABLE Frames are used by endpoints to inform the peer
+that the path associated to
+the Destination Connection IDs in the frame is available for sending.
+PATH_AVAILABLE frames are formatted as shown in {{fig-path-available-format}}.
+
+~~~
+  PATH_AVAILABLE Frame {
+    Type (i) = TBD-03 (experiments use 0xbaba07),
```

Comment:
We should update the base value, and 07 is already taken (because 06 was the PATH_STATUS frame from draft 05)
```suggestion
    Type (i) = TBD-03 (experiments use 0x15228c08),
```